mix.exs

```diff
@@ -17,18 +17,25 @@ defmodule OpenIDConnect.Mixfile do
       deps: deps(),
       docs: docs(),
       name: "OpenID Connect",
-      source_url: "https://github.com/DockYard/openid_connect"
+      source_url: "https://github.com/firezone/openid_connect",
```

Comment:
this needs to be restored to the DockYard one

Reply:
✅